tag.php

```diff
@@ -16,103 +16,152 @@
 use P4BKS\Controllers\Blocks\CampaignThumbnail_Controller as CampaignThumbnail;
 use P4BKS\Controllers\Blocks\HappyPoint_Controller as HappyPoint;
 
-$templates = [ 'tag.twig', 'archive.twig', 'index.twig' ];
-
 $context = Timber::get_context();
 
 if ( is_tag() ) {
 	$context['tag']  = get_queried_object();
 	$explore_page_id = planet4_get_option( 'explore_page' );
 
-	$posts = get_posts(
-		[
-			'posts_per_page'   => 1,
-			'offset'           => 0,
-			'post_parent'      => $explore_page_id,
-			'post_type'        => 'page',
-			'post_status'      => 'publish',
-			'suppress_filters' => false,
-			'tag_slug__in'     => [ $context['tag']->slug ],
-		]
-	);
-
-	$context['custom_body_classes'] = 'white-bg page-issue-page';
-	$context['category_name']       = $posts[0]->post_title ?? '';
-	$context['category_link']       = isset( $posts[0] ) ? get_permalink( $posts[0] ) : '';
-	$context['tag_name']            = single_tag_title( '', false );
-	$context['tag_description']     = wpautop( $context['tag']->description );
-	$context['tag_image']           = get_term_meta( $context['tag']->term_id, 'tag_attachment', true );
-	$tag_image_id                   = get_term_meta( $context['tag']->term_id, 'tag_attachment_id', true );
-
-	$context['og_description'] = $context['tag_description'];
-	if ( $tag_image_id ) {
-		$context['og_image_data'] = wp_get_attachment_image_src( $tag_image_id, 'full' );
-	}
+	$redirect_id = get_term_meta( $context['tag']->term_id, 'redirect_page', true );
+	if ( $redirect_id ) {
+
+		$redirect_page = get_post( $redirect_id );
```

Comment:
Could you try this? to avoid duplicating the page template (although it's hack also) 😰 
```
global $wp_query;
$wp_query->queried_object = $redirect_page;
$wp_query->queried_object_id = $redirect_page->ID;
include 'page.php';
```

Reply:
I amended the commit with this suggestion and it seems to work fine. It's definitely more clean that the original approach.